### DISCUSSION TITLE
Engine Detection

### DISCUSSION THREAD

**Discussion by dalogue1** [Nachtzuster/BirdNET-Pi#109] [General]
For some reason my system has started identifying something called Engine.

This is obviously meaningless but what is this caused by?

21:56:34---[server][INFO] READING AUDIO DATA...
21:56:34---[server][INFO] READING DONE! READ 5 CHUNKS.
21:56:34---[server][INFO] ANALYZING AUDIO...
21:56:35---[server][INFO] DONE! Time 0.51 SECONDS
21:56:35---[server][INFO] 0.0;3.0-('Human_Human', 0.0)
21:56:35---[server][INFO] 3.0;6.0-('Engine_Engine', 0.4774843)
21:56:35---[server][INFO] 6.0;9.0-('Engine_Engine', 0.57757336)
21:56:35---[server][INFO] 9.0;12.0-('Human_Human', 0.0)
21:56:35---[server][INFO] 12.0;15.0-('Engine_Engine', 0.6450318)
[
<img width="1194" alt="Screen Shot 2024-06-16 at 10 01 12 PM" src="https://github.com/Nachtzuster/BirdNET-Pi/assets/133064876/8aabd568-7a67-4537-bc0a-1a4cc7c97891">
](url)

**Comment by dalogue1** [comment]
It started when I changed the Privacy Threshold to 0% and went away when I changed it back to 1%.

Again, what does Engine mean?

**Reply by lloydbayley** [reply]
By the look of the noise in the spectrogram, It thinks it has detected an idling engine!

**Comment by dalogue1** [comment]
It is next to my A/C units.  Maybe that is it.

**Comment by lloydbayley** [comment]
i'd say that it would be. I think this can be marked as Answered now, yes?

**Comment by dalogue1** [comment]
yes
